.prettierignore

```diff
@@ -8,7 +8,8 @@ coverage/
 static
 _build
 # File types which Prettier supports but we don’t want auto-formatting.
-*.md
+.github/
```

Comment:
I’m not sure about this one. Perhaps we should format it all by default and use `<!-- prettier-ignore -->` comments for parts of the files that have very specific formatting?

Reply:
It will be very weird to see that comment in the pull request and issue templates though

Reply:
I can just ignore the one specific file that we have added multiple line-breaks to

Reply:
I also tried the comment and it did not seem to work. 
Used syntax as per https://prettier.io/docs/en/ignore.html#markdown